lib/order.rb

```diff
@@ -0,0 +1,25 @@
+class Order
+  attr_reader :items
+  def initialize menu
+    @menu  = menu
+    @items = {}
+  end
+
+  def add dish
+    if items.has_key? dish
+      add "#{dish}*".to_sym
+    else
+      items[dish] = @menu.price_of dish # coupled to menu
+    end
+    "Subtotal: £#{total}."
+  end
+
+  def confirm message_centre, customer_details
+    "Total: £#{total}."
```

Comment:
This won't ever print b/c it is not the return value - also it is a weird return value